lib/determine-basal/determine-basal.js

```diff
@@ -102,15 +102,15 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     // adjust min, max, and target BG for sensitivity, such that 50% increase in ISF raises target from 100 to 120
     if (typeof autosens_data !== 'undefined' && profile.autosens_adjust_targets) {
       if (profile.temptargetSet) {
-        console.error("Temp Target set, not adjusting with autosens");
+        process.stderr.write("Temp Target set, not adjusting with autosens; ");
       } else {
         min_bg = round((min_bg - 60) / autosens_data.ratio) + 60;
         max_bg = round((max_bg - 60) / autosens_data.ratio) + 60;
         new_target_bg = round((target_bg - 60) / autosens_data.ratio) + 60;
         if (target_bg == new_target_bg) {
-            console.error("target_bg unchanged:", new_target_bg);
+            process.stderr.write("target_bg unchanged: "+new_target_bg+"; ");
```

Comment:
You're missing the context on these.  The point of process.stderr.write instead of console.error is I'm switching them to be inline with previous ones to save screen space.